config/settings/production.py

```diff
@@ -255,9 +257,24 @@
 
 HEROKU_TOKEN = env("HEROKU_TOKEN", default=None)
 HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
+
 if HEROKU_TOKEN and HEROKU_APP_NAME:
     METACI_WORKER_AUTOSCALER = "metaci.build.autoscaling.HerokuAutoscaler"
 
+# Autoscalers are defined per METACI_APP
+AUTOSCALERS = json.loads(env("AUTOSCALERS", default={}))
+
+if not AUTOSCALERS:
+    # backwards compatability
+    HEROKU_APP_NAME = env("HEROKU_APP_NAME", default=None)
+    AUTOSCALERS = {
+        HEROKU_APP_NAME: {
+            "app_name": HEROKU_APP_NAME,
+            "worker_type": "worker",
```

Comment:
```suggestion
            "worker_type": WORKER_DYNO_NAME,
```

for backwards-compatibility in case someone changed that setting